#include "scene_ms_player.h"
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE

namespace msplayerservice{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE

///<<<rpc begin
void PlayerPlayerSceneServiceImpl::EnterScene(common::EntityPtr& entity,
    const msplayerservice::SeceneTestRequest* request,
    msplayerservice::SeceneTestResponse* response)
{
///<<< BEGIN WRITING YOUR CODE EnterScene
///<<< END WRITING YOUR CODE EnterScene
}

void PlayerPlayerSceneServiceImpl::EnterSceneNotify(common::EntityPtr& entity,
    const msplayerservice::EnterSeceneS2C* request,
    msplayerservice::EnterSeceneS2C* response)
{
///<<< BEGIN WRITING YOUR CODE EnterScene
///<<< END WRITING YOUR CODE EnterScene
}

///<<<rpc end
///<<< BEGIN WRITING YOUR CODE EnterSceneNotify
///<<< END WRITING YOUR CODE EnterSceneNotify

}// namespace msplayerservice
